routing/graph.go

```diff
@@ -26,33 +26,25 @@ type routingGraph interface {
 // database.
 type dbRoutingTx struct {
 	graph  *channeldb.ChannelGraph
-	tx     kvdb.RTx
 	source route.Vertex
 }
 
-// newDbRoutingTx instantiates a new db-connected routing graph. It implictly
-// instantiates a new read transaction.
+// newDbRoutingTx instantiates a new db-connected routing graph.
```

Comment:
>To make sure we don't interfere with the kvdb logic when it comes to remote backends such as etcd, we use the kvdb.View method instead of keeping a long-lived read TX open.

What kind of interference is this exactly, a dead lock?

Reply:
Okay, I think I drop the commit for the moment since we have identified the actual culprit that breaks the itests. We'll need to address the path finding later on anyway.